FirstProject/new-evaluation.html

```diff
@@ -56,29 +57,29 @@ <h2 class="description_section">Technical level</h2>
         <div class="row_section">
             <div class="col-md-3 bullets_menu">
                 <form action="">
-                    <input type="radio" name="circle" value="circles">
+                    <input class="input-circle" type="radio" name="circle" value="circles">
```

Comment:
I try to use input-circle class to style the circles. I use more inputs with same class because are places where I need for example 3 inputs. And then I use same class for all three inputs.